script.js

```diff
@@ -0,0 +1,40 @@
+const btn = document.querySelector('#btn');
+const ti = document.querySelector('#title');
+const au = document.querySelector('#author');
+const book = document.querySelector('#content');
+const data = JSON.parse(localStorage.getItem('bookData')) || [];
+
+btn.addEventListener('click', () => {
+  const item = {
+    title: ti.value,
+    author: au.value,
```

Comment:
- [ ] [optional] @Shakir-Hussain12 and @Ezzywealth it would have been nice if you added a unique id property for each book since it would make it easier to remove books with duplicate values

Reply:
thanks, will do that in the next PR for sure.